.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: Checkout sources
         uses: actions/checkout@v3
 
-      - run: rustup toolchain install stable --profile minimal --component rustfmt --component clippy --no-self-update
```

Comment:
We want the *latest* stable version whenever that is available. We don’t want to be at the mercy of whenever github feels like updating their Runners.

Reply:
They have weekly cadence though, so not sure if it's that crucial tbh.
If we want to update it though, we'll need to change the default everywhere as well, as it's tied to version installed in the image.